tests/test.py

```diff
@@ -17,55 +78,16 @@ def __init__(self, *args, **kwargs):
 
         super(TestParser, self).__init__(*args, **kwargs)
 
-    @parameterized.expand([
-        ('grapple', 50080840),
-        ('test', 50342756),
-        ('patronise', 49023308),
-        ('abiologically', 43781266),
-        ('alexin', 50152026),
-        ('song', 50235564),
-        ('house', 50356446),
-    ])
-    def test_words_from_english(self, word: str, old_id: int):
-        self.__test_word(word, old_id, 'English')
-
-    @parameterized.expand([
-        ('video', 50291344),
-    ])
-    def test_words_from_latin(self, word: str, old_id: int):
-        self.__test_word(word, old_id, 'Latin')
-
-    @parameterized.expand([
-        ('seg', 50359832),
-        ('aldersblandet', 38616917),
-        ('by', 50399022),
-        ('for', 50363295),
-        ('admiral', 50357597),
-        ('heis', 49469949),
-        ('konkurs', 48269433),
-        ('pantergaupe', 46717478),
-        ('maldivisk', 49859434),
-    ])
-    def test_words_from_norwegian_bokmal(self, word: str, old_id: int):
-        self.__test_word(word, old_id, 'Norwegian Bokmål')
-
-    @parameterized.expand([
-        ('house', 50356446)
-    ])
-    def test_words_from_swedish(self, word: str, old_id: int):
-        self.__test_word(word, old_id, 'Swedish')
-
-    @parameterized.expand([
-        ('ἀγγελία', 47719496)
-    ])
-    def test_words_from_ancient_greek(self, word: str, old_id: int):
-        self.__test_word(word, old_id, 'Ancient Greek')
-
-    def __test_words(self, words_and_ids: Dict[str, int], lang: str):
-        for word, old_id in words_and_ids.items():
-            self.__test_word(word, old_id, lang)
-
-    def __test_word(self, word: str, old_id: int, lang: str):
+    @parameterized.expand(get_test_words_table())
+    @mock.patch("requests.Session.get", side_effect=mocked_requests_get)
+    def test_fetch_using_mock_session(self, lang: str, word: str, old_id: int, mock_get):
+        self.__test_fetch(lang, word, old_id)
+
+    @parameterized.expand(get_test_words_table('ἀγγελία', 'test'))
+    def test_fetch_using_actual_session(self, lang: str, word: str, old_id: int):
```

Comment:
I'd say let's not make actual HTTP calls in the tests at all. We don't want to run into the same issue later where the page structure changes in the future and some of the tests start failing